File: mvn-advent-of-code/src/test/java/advent/twenty_sixteen/Day01NoTimeForATaxicabTest.java
package advent.twenty_sixteen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day01NoTimeForATaxicabTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day01NoTimeForATaxicab("/twenty_sixteen/Day01-NoTimeForATaxicab-input.txt");

        assertEquals(301, dp.getPart1Answer().intValue());
        assertEquals(130, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day01NoTimeForATaxicab("/twenty_sixteen/Day01-NoTimeForATaxicab-example#1.txt");

        assertEquals(8, dp.getPart1Answer().intValue());
        assertEquals(4, dp.getPart2Answer().intValue());
    }
}

File: kotlin-advent-of-code/src/test/kotlin/advent/twenty_sixteen/Day01NoTimeForATaxicabTest.kt
package advent.twenty_sixteen

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class Day01NoTimeForATaxicabTest {
    @Test
    fun checkReal() {
        val dp = Day01NoTimeForATaxicab("/twenty_sixteen/Day01-NoTimeForATaxicab-input.txt")

        Assertions.assertEquals(301, dp.part1Answer)
        Assertions.assertEquals(130, dp.part2Answer)
    }

    @Test
    fun checkExample1() {
        val dp = Day01NoTimeForATaxicab("/twenty_sixteen/Day01-NoTimeForATaxicab-example#1.txt")

        Assertions.assertEquals(8, dp.part1Answer)
        Assertions.assertEquals(4, dp.part2Answer)
    }
}